js/build.js

```diff
@@ -1,45 +1,55 @@
-(function(){
-  window.ui = window.ui || {}
+(function() {
+  window.ui = window.ui || {};
   ui.flipletCharts = ui.flipletCharts || {};
 
   Fliplet.Chart = Fliplet.Widget.Namespace('chart');
 
   function init() {
-    Fliplet.Widget.instance('chart-pie-1-1-0', function (data) {
+    Fliplet.Widget.instance('chart-pie-1-1-0', function(data) {
       var chartId = data.id;
-      var chartUuid = data.uuid;
       var $container = $(this);
+      var themeInstance = Fliplet.Themes.Current.getInstance();
+      var themeValues = themeInstance.data.values ? Object.assign({}, themeInstance.data.values) : {};
+
+      if (themeInstance.data.hasOwnProperty('widgetInstances') && themeInstance.data.widgetInstances.length) {
+        themeInstance.data.widgetInstances.forEach(function(widgetProp) {
```

Comment:
Use `_.forEach` then you don't need the `if` clause because lodash manages undefined inputs gracefully.

```suggestion
       _.forEach(themeInstance.data.widgetInstances, function(widgetProp) {
```

Reply:
Thank you, I'll pick up your corrections